pyworklistserver/worklist.py

```diff
@@ -1,108 +1,170 @@
-            worklist_item.PatientName = self._get_person_name()
-            worklist_item.PatientID = _extend_with_random_to_length('Patient id ', 64)
-            worklist_item.IssuerOfPatientID = _extend_with_random_to_length('Issuer of patient id ', 64)
-            worklist_item.PatientWeight = str(random.uniform(10.0, 150.0))[:16]
-            worklist_item.PatientSize = str(random.uniform(1.0, 2.5))[:16]
-            worklist_item.AdmissionID= _extend_with_random_to_length('Admission id ', 64)
-            worklist_item.RequestedProcedureID = _extend_with_random_to_length('Step id ', 16)
-            worklist_item.RequestedProcedureDescription = _extend_with_random_to_length('Step description ', 64)
-
-            otherPatientIdsSq = [Dataset(), Dataset()]
-            for otherPatientId in otherPatientIdsSq:
-                otherPatientId.PatientID = _extend_with_random_to_length('Other patient id ', 64)
-                otherPatientId.IssuerOfPatientID = _extend_with_random_to_length('Issuer of patient id ', 64)
-                otherPatientId.TypeOfPatientID = 'TEXT'
-
-            worklist_item.OtherPatientIDsSequence = otherPatientIdsSq
-
-            step = Dataset()
-            step.ScheduledPerformingPhysicianName = self._get_person_name()
-            step.ScheduledProcedureStepStartDate = _random_dicom_date_after_1900()
-            step.ScheduledProcedureStepStartTime = _random_dicom_time()
-            step.ScheduledProcedureStepDescription = _extend_with_random_to_length('Scheduled procedure step desc ', 64)
-            step.CommentsOnTheScheduledProcedureStep = _extend_with_random_to_length('Scheduled step comments ', 10240)
-            worklist_item.ScheduledProcedureStepSequence = [step]
-
-            # Other patient ID sequence patientid issuer of patient id
-
-            worklist.append(worklist_item)
-        return worklist
-
-    def _get_person_name(self):
-        """ Create a random person name and truncate the name components according to Vivid bug """
-        return _random_person_name(32)[:_VIVID_HACK_MAX_PERSON_NAME]  # fix for wrong handling in EchoPAC/Scanner
-    
\ No newline at end of file
+""" Worklist generator classes """
+
+import datetime
+import random
+import string
+from pydicom.uid import generate_uid
+from pydicom.dataset import Dataset
+from pyworklistserver import fault_provider
+from pyworklistserver import user_config
+
+
+__NONASCII = 'æÆøØåÅßäöüÄÖÜ' #just an arbitrarily selected list of non ascii characters
+
+__CHINESE = '也池馳弛水马弓土人女' #An excempt of chinese characters
+
+__RUSSIAN = 'ДРЛИПЦЗГБЖ'   #An excempt of russian characters
+
+__GREEK = 'ΑαΒβΓγΔδΕεΖζΗηΘθΙιψΩω' #An excempt of Greek characters
+
+__JAPANESE = '日一大二目五後.女かたまやたば' #An excempt of Japanese characters (Kanji, Hiragana and Katakana)
+
+__KOREAN = 'ㄱㄴㄷㄹㅇㅈㅑㅓㅕㅗㅛㅔㅖㅚㅿㆆㆍ' #An excempt of Korean characters (Hangul)
+
+def _random_unicode_string(length):
```

Comment:
This function is getting too many responsibilities. Maybe it should take the language specific characters as input. This way, it is up to the caller to decide which language to select from. Then, there could be a separate function responsible for randomly selecting language.